src/server/lib/okta/fixProfile.ts

```diff
@@ -31,7 +31,7 @@ export const fixOktaProfile = async ({
 		await updateUser(oktaId, {
 			profile: {
 				legacyIdentityId: idapiUser.id,
-				searchPartitionKey: sha256(idapiUser.id),
+				searchPartitionKey: sha256Hex(idapiUser.id),
```

Comment:
what values was this generating before ? if there are values that are not valid hex then the user will be excluded from the export

Reply:
As discussed in standup, these were accidentally generating base64 encoded values, where they should've been in hex. @raphaelkabo wrote a script to backfill users affected by this.

Reply:
for future reference: we talked about this during standup there's a script being run to fix the users with the search partition in the wrong format